features : filter and sort bottom sheet added

AbhimanyuT0800 opened the pull request:
- filter and sort bottom sheet added in search/widgets folder
- filter and sort constants rearranged and folder renamed
- utils: bottom sheet updated

AbhimanyuT0800 committed:
utils : bottom sheet upadated,default border radius updated

AbhimanyuT0800 committed:
filter tabbar added

AbhimanyuT0800 committed:
filter bottom sheet basic ui complete

AbhimanyuT0800 committed:
features: Filter&Sort added

- Filter and sort added in search/widgets folder
- bottom sheet updated
- filter and sort constants rearrange

AbhimanyuT0800 committed:
order and sort bottom sheet changed

AbhimanyuT0800 committed:
Merge branch 'main' of https://github.com/LX-PROJECTS/renthub into filter-bottom-sheet

AbhimanyuT0800 committed:
widgets refactorized

AbhimanyuT0800 committed:
Delete lib/features/search/widgets/filter_bottom_sheet_widget.dart

Sireen-Official commented on lib/features/search/widgets/filter_widgets/filter_sctn_widget.dart:
Keep only one widget in one file

AbhimanyuT0800 committed:
features: filter and sort widgets changed

- functions changed to widgets
- bottom nav and filter sort contants class name and provider name
  renamed
-

AbhimanyuT0800 committed:
Merge branch 'filter-bottom-sheet' of https://github.com/LX-PROJECTS/renthub into filter-bottom-sheet
